docs/mobile-apps/automated-testing/espresso-xcuitest/real-devices.md

```diff
@@ -139,35 +152,42 @@ You can use any of the existing methods of distribution for your iOS app, except
 3. Compress the Payload directory into an archive (.zip file) and give it a new name with .ipa file format ending.
 4. The .ipa file is ready to be used in the `--test` parameter of our XCUITest test runner.
 
+#### Configuring XCUITest Files for Sauce Runner
```

Comment:
I'm not 100% sure what the diff is between this chapter and the things that are explained above, this seems like double info

Reply:
Agreed! I've made some changes to this section and a related doc about creating .ipa files. I'll send to you separately for review. I found it hard to distinguish which instructions were for Appium vs. XCUITest.